SlackAPI/Message.cs

```diff
@@ -22,6 +22,6 @@ public class Message : SlackSocketMessage
         public Reaction[] reactions;
         //Wibblr? Not really sure what this applies to.  :<
 
-        public DateTime? thread_ts;
+        public string thread_ts;
```

Comment:
We are open to suggestions here but the conversion the DateTime type was causing some fidelity loss and the conversation reply api needs an exact timestamp.